src/lib/layout/Header.tsx

```diff
@@ -1,43 +1,93 @@
 import { Flex, Image } from "@chakra-ui/react";
+import type { ReactNode } from "react";
 
-import { useCelatoneApp } from "lib/app-provider";
+import { useCelatoneApp, useInitia } from "lib/app-provider";
 import { AppLink } from "lib/components/AppLink";
 import { FaucetBtn } from "lib/components/button";
 import { WalletSection } from "lib/components/Wallet";
 
+import { AppMenu } from "./AppMenu";
 import { NetworkMenu } from "./NetworkMenu";
 import Searchbar from "./Searchbar";
 
+export const SectionWrapper = ({
```

Comment:
Create file for this component and don't forget to declare `SectionWrapperProps`